core/index.py

```diff
@@ -666,3 +666,15 @@ def _solrize_date(d, is_start=True):
     else:
         return None
 
+def get_page_text(page):
+   no_text = ["Text not available"]
+   solr = SolrConnection(settings.SOLR)
+   title, date = page.issue.title, page.issue.date_issued
+   query = 'title:"' + str(title) + '" and date:' + str(date).replace('-', '') + \
+           ' and sequence:' + str(page.sequence)
+   solr_results = solr.query(query)
+   results_attribute = getattr(solr_results, 'results', None)
+   if results_attribute and len(results_attribute) > 0:
```

Comment:
This test is redundant as written – both `None` and `[]` will fail the `if results_attribute` check. If the idea is to test whether it's a value other than either `None` or a `list`, I would do an `isinstance` check instead.

Reply:
added isinstance(results_attribute, list)